src/actions/NotificationActions.js

```diff
@@ -44,7 +46,10 @@ export const registerNotificationsV2 = () => async (dispatch: Dispatch, getState
   try {
     const body = {
       apiKey: ENV.AIRBITZ_API_KEY,
-      deviceId: state.core.context.clientId
+      deviceId: state.core.context.clientId,
```

Comment:
Same problem here. We need to provide both `loginIds` and `deviceToken` in the top-level body. Without the `deviceToken`, the query will actually de-register notifications for this device.